www/wrap-root-element.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { MDXProvider } from "@mdx-js/react"
+import GuideList from "./src/components/guide-list.js"
+import HubspotForm from "./src/components/hubspot-form"
+import Pullquote from "./src/components/shared/pullquote"
+import DateChart from "./src/components/chart"
+
+const components = {
+  GuideList,
+  HubspotForm,
+  DateChart,
+  Pullquote,
+}
+
+export default ({ element }) => (
+  <MDXProvider components={components}>{element}</MDXProvider>
```

Comment:
These all get loaded to the global app.js? How much bigger does app.js get with this change?

Reply:
I don't know. I wasn't able to check build times or anything either because sharp was breaking the build every time I tried to run the site :/

Reply:
@ChristopherBiscardi are you still getting the build error? Maybe you could try again?